src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/sykefraværshistorikk/summert/SummertKorttidsOgLangtidsfravær.java

```diff
@@ -74,6 +74,7 @@
         );
     }
 
+    // TODO: bruk SummertSykefravær.getSummertSykefravær() i stedet
```

Comment:
Var dette ment å bli gjort nå?

Reply:
Tror at det er markert for å bli gjort senere. 
Sånn bare påminnelse.

Reply:
Fjernet kommentar